recipes/cylc/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "cylc" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  # Skip as Cylc requires Linux
+  skip: True  # [py<37 or not linux]
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - 'configurable-http-proxy >=4.1'
+    - cylc-flow ==8.0a1
```

Comment:
Seems like this should be `cylc-flow =={{ version }}`

Reply:
Oh, good point! Updating it now. Thanks!

Reply:
Done!